Tags/hacienda.json

```diff
@@ -0,0 +1,46 @@
+{
+"hate": [],
+"love": [
```

Comment:
This file needs to be

```json
[
  "/Dancer/Blobby_Mirror/LuxXx-BadBallz",
  "/Dancer/Blobby_Mirror/LuxXx-Dancing_Or_Mating_Blobs_II",
  "..."
]
```

That is just an array, not the object with `hate` and `love` arrays.

Sorry, I can circle back to the clipboard copy/paste tool to make it match this format soon.

Reply:
also fixing